test/dotnet-new.Tests/NewCommandTests.cs

```diff
@@ -19,8 +16,7 @@ public void WhenSwitchIsSkippedThenItPrintsError()
 
             cmd.ExitCode.Should().NotBe(0);
             
-			cmd.StdErr.Should().Be("Unrecognized command or argument 'Web1.1'");
-            cmd.StdOut.Should().Be("Specify --help for a list of available options and commands.");
```

Comment:
I'm adding another similar test. Depending on the way the specified template doesn't match, there are 2 possible error paths that can be taken. This test demonstrates one of them, the test I'll be checking in demonstrates the other one.